src/utils/isCheck.ts

```diff
@@ -0,0 +1,18 @@
+import { PrivilegeLevel } from '../auth/ducks/types';
+import { WindowTypes } from '../components/windowDimensions';
+
+/**
+ * Checks if the user is logged in
+ * @param privilegeLevel the user's privilege level
+ */
+export function isLoggedIn(privilegeLevel: PrivilegeLevel): boolean {
```

Comment:
🤔 I wonder if this might be better as a selector on auth? Maybe not but just a thought

Reply:
Good call